### ISSUE TITLE
Fitting AutoARIMA on constant time series causes TypeError

### ISSUE THREAD

**Issue by shagn** [Nixtla/statsforecast#121]
**Version**: v0.5.5

**Description**
When fitting an instance of AutoARIMA with default parameters on a constant time series, the Arima function gets called with the keyword argument "fixed", but this argument is not specified in the function. 

**To Reproduce**
```
from statsforecast.arima import AutoARIMA
import numpy as np

AutoARIMA().fit(np.array([1]*36))
```

**Expected behaviour**
AutoARIMA should call the Arima function only with available arguments (see function signature below): 
```python
def Arima(
    x,
    order=(0, 0, 0),
    seasonal={'order': (0, 0, 0), 'period': 1},
    xreg=None,
    include_mean=True,
    include_drift=False,
    include_constant=None,
    blambda=None,
    biasadj=False,
    method='CSS',
    model=None,
)
```

**Screenshots**
![image](https://user-images.githubusercontent.com/16029092/171033805-c5e73aeb-67b5-4502-8996-f166c99f7597.png)

**Comment by FedericoGarza** [comment]
Hi @shagn! Thanks for catching that. We fixed the issue in #122. It's now in the main branch. We will release a new version soon. 

<img width="473" alt="image" src="https://user-images.githubusercontent.com/10517170/171059153-a0dea561-5843-434a-b30c-b1beaa577010.png">

**Comment by mergenthaler** [comment]
@all-contributors please add @shagn for bug
